modules/vf-graphql-holochain/resolvers/plan.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Resolvers for EconomicEvent fields
+ *
+ * @package: HoloREA
+ * @since:   2019-08-27
+ */
+
+import { DNAIdMappings, DEFAULT_VF_MODULES, VfModule } from '../types'
+import { extractEdges, mapZomeFn } from '../connection'
+
+import {
+  Process,
+  PlanProcessFilterParams
+} from '@valueflows/vf-graphql'
+
+
+export default (enabledVFModules: VfModule[] = DEFAULT_VF_MODULES, dnaConfig: DNAIdMappings, conductorUri: string) => {
+
+  const hasAgent = -1 !== enabledVFModules.indexOf(VfModule.Agent)
+  const hasKnowledge = -1 !== enabledVFModules.indexOf(VfModule.Knowledge)
+  const hasPlanning = -1 !== enabledVFModules.indexOf(VfModule.Planning)
+  const hasAgreement = -1 !== enabledVFModules.indexOf(VfModule.Agreement)
+
+  const readProcesses = mapZomeFn(dnaConfig, conductorUri, 'observation', 'process_index', 'query_processes')
+
+  return Object.assign(
```

Comment:
I'm imagining we would add a resolver for the `inScopeOf` field once `Person` & `Organization` are implemented (because `inScopeOf` is of type `[AccountingScope!]`

Reply:
Correct. And we might end up simplifying `Agent` a lot by just having person/org as a type field, so `AccountingScope` might only ever be an (REA) Agent for our purposes.